catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/MiroTransformableTransformerTest.scala

```diff
@@ -273,6 +273,12 @@ class MiroTransformableTransformerTest
     work.items.head.agent.locations shouldBe List(expectedDigitalLocation)
   }
 
+  it("sets the WorkType as 'Digital images'") {
+    val work = transformWork()
+    work.workType.isDefined shouldBe true
+    work.workType.get.label shouldBe "Digital images"
```

Comment:
maybe
`work.workType shouldBe defined`
or
`work.workType shouldBe Some(WorkType("q", "Digital images"))`?

not sure if the `id` matters

Reply:
It's sort of arbitrary that the ID is "q" – it's cribbed from the Sierra work types spreadsheet.

I didn't want to write something that was a literal clone of the code under test.

I don't think it makes much of a difference.

Reply:
Sure, if it doesn't matter don't test for it.